CGFloatType.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |spec|
   spec.name = 'CGFloatType'
-  spec.version = '1.1.0'
+  spec.version = '1.1.1'
```

Comment:
This will be 1.2.0 because it introduces new API ([semantic versioning](http://semver.org)).